web/styles/settings.css

```diff
@@ -1321,40 +1319,37 @@ label.preferences-radio-choice-label {
 
         .tab-container {
             box-sizing: border-box;
-            height: var(--settings-header-height);
-            padding: 6px;
+            height: var(--settings-overlay-header-height);
+            display: flex;
+            align-items: center;
+            justify-content: center;
             background-color: var(--color-background-modal-bar);
             border-bottom: 1px solid var(--color-border-modal-bar);
 
-            @media (width >= $md_min) {
-                .tab-switcher {
-                    position: absolute;
-                    display: flex;
-                    left: 10px;
-                    z-index: 1;
-
-                    .ind-tab {
-                        width: auto;
-                        min-width: 95px;
-                    }
+            .tab-switcher {
+                display: flex;
+
+                .ind-tab {
+                    width: auto;
                 }
             }
 
-            @media (width < $md_min) {
+            @container (width >= $settings_overlay_sidebar_collapse_breakpoint) {
+                .tab-switcher .ind-tab {
+                    min-width: 6.7857em; /* 95px at 14px em */
```

Comment:
Would there be any harm in moving this `min-width` value out of a container query? possibly with the `justify-content` below? How is the content justified outside of the smaller width?

Reply:
I tried both of these rules at both widths and it seems fine/better to have them for both! Thanks for noticing.